src/components/SortOrderDropdown.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+import { Dropdown } from "semantic-ui-react";
+
+import "semantic-ui-css/semantic.min.css";
+
+class SortOrderDropdown extends Component {
+  constructor(props) {
+    super(props);
+  }
```

Comment:
Please remove these to clear warning.
```
./src/components/SortOrderDropdown.js
  Line 7:3:  Useless constructor  no-useless-constructor
```
 Ref: https://eslint.org/docs/rules/no-useless-constructor